hoa2dot/core.py

```diff
@@ -80,6 +93,9 @@ def accepting_sets(self) -> Set[int]:
     def __str__(self):
         return " ".join(map(str, self.conditions))
 
+    def __eq__(self, other):
+        return isinstance(other, Or) and self.conditions == other.conditions
```

Comment:
please add docstring, o/w `flake8` test won't pass.

Reply:
flake8 passed fine

Reply:
The same holds for all the `__eq__` and `__str__` etc.

Reply:
ok, I'll put docstring for all